index.js

```diff
@@ -71,43 +60,39 @@ class MongoModels {
     }
 
 
-    static count() {
+    static count(...args) {
 
-        const args = argsFromArguments(arguments);
         const db = dbFromArgs(args);
         const collection = db.collection(this.collectionName);
 
-        return collection.count.apply(collection, args);
+        return collection.countDocuments(...args);
```

Comment:
Since `collection.count` isn't being deprecated and offers slightly different functionality (uses meta data instead of aggregating). Let's revert this and create a new method `countDocuments` instead.

Reply:
I was wrong. `collection.count` is being deprecated.